kloppy/domain/services/transformers/dataset.py

```diff
@@ -245,23 +239,20 @@ def __change_point_coordinate_system(
         if not point:
             return None
 
-        x = self._from_pitch_dimensions.x_dim.to_base(point.x)
-        y = self._from_pitch_dimensions.y_dim.to_base(point.y)
+        point_base = self._from_pitch_dimensions.to_base(point)
 
         if (
             self._from_coordinate_system.vertical_orientation
             != self._to_coordinate_system.vertical_orientation
         ):
-            y = 1 - y
+            point_base = replace(
+                point_base,
+                y=68 - point_base.y,
```

Comment:
This seems like a bit of a magic number here. Can it be moved to some other place or to a 'constant' ?

Reply:
I've replaced it by a constant + optimized it a bit more to avoid unneeded conversions between dimensions.